app/templates/components/github-apps-repository.hbs

```diff
@@ -8,6 +8,12 @@
   {{this.name}}
 </LinkTo>
 {{#if this.admin}}
+  {{#if this.isNotMatchGithub}}
+    <TravisSwitch
+      @active={{this.repository.active}}
+      @onToggle={{perform this.toggleRepositoryTask}}
+    />
+  {{/if}}
```

Comment:
I'm not sure when this would come up, it seems like this component is only used when `isGithubRepository` is true. But maybe there's a case I'm forgetting about, or this is precautionary. Either way, I think this is fine, just wanted to note and ask about it :smile:

Reply:
Actually github-apps-repository component is used for all repositories, not only GH ones. You are right this is a little confusing name. We should refactor it (at least rename) in future.